Controller/MediaAdminController.php

```diff
@@ -109,6 +118,132 @@ public function listAction(Request $request = null)
     }
 
     /**
+     * @throws \Symfony\Component\Security\Core\Exception\AccessDeniedException|\LogicException
+     *
+     * @return \Symfony\Bundle\FrameworkBundle\Controller\Response|\Symfony\Component\HttpFoundation\Response
+     */
+    public function multiUploadAction(Request $request)
+    {
+        if (false === $this->admin->isGranted('CREATE')) {
```

Comment:
This always returns a boolean. Just write `if (!$this->admin->isGranted('CREATE'))`